cristin-commons/src/main/java/no/unit/nva/cristin/model/Constants.java

```diff
@@ -62,6 +62,8 @@ public class Constants {
     public static final String HASHTAG = "#";
     public static final String SLASH_DELIMITER = "/";
     public static final String CRISTIN_INSTITUTION_HEADER = "Cristin-Representing-Institution";
+    public static final String BAD_BOT_BYPASS = "UiO-BadBotBypass";
+    public static final String LET_ME_IN = "LetMeIn";
```

Comment:
Is it okay putting them here or should we use env vars from github?

Reply:
I would have used an env, but now you need to change the "secret" since it is in git